src/navigation/constant_curvature_arcs.cc

```diff
@@ -64,6 +68,19 @@ void ConstantCurvatureArc::GetControls(const MotionLimits& linear_limits,
   ang_vel_cmd = vel_cmd.x() * curvature;
 }
 
+Pose2Df ConstantCurvatureArc::GetIntermediateState(float f) const {
+  const float a = Sign(curvature) * angular_length * f;
+  if (length == 0) {
+    // Pure rotational motion.
```

Comment:
Won't `a` be nan in that case? curvature for pure rotational motion is inf.

Reply:
ah, interesting. I don't think this case is being hit  anyways, since length is always > 0 for the paths we consider. I can just take out this if condition, unless you have a better way to handle this case, or think it needs to be handled here